cmd/scollector/collectors/collectors.go

```diff
@@ -266,7 +258,20 @@ func AddTS(md *opentsdb.MultiDataPoint, name string, ts int64, value interface{}
 		Value:     value,
 		Tags:      tags,
 	}
-	*md = append(*md, &d)
+	if d.Valid() {
```

Comment:
Better to:
```
if !d.Valid() {
   slog.Errorf(...)
   return
}
```

That way the rest of the logic doesn't have to be indendented. See https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow